scripts/build-npm.sh

```diff
@@ -20,7 +20,7 @@ verify_star_dependencies() {
 }
 
 # Find and build all NPM projects
-for pkgJson in $(find typescript -name package.json | grep -v node_modules); do
+for pkgJson in $(find typescript -name cdk.json | grep -v node_modules); do
```

Comment:
I've run the following commands to compare the directories that are evaluated:
```
find typescript -name package.json  | grep -v node_modules | xargs -n 1 dirname | sort
find typescript -name cdj.json  | grep -v node_modules | xargs -n 1 dirname | sort
```

Here is the diff. Directories removed are shown in grey on the left-hand side:
![Screen Shot 2020-04-18 at 5 50 34 pm](https://user-images.githubusercontent.com/2453371/79673590-02022900-81d3-11ea-9f22-d7eb87153230.png)
[Link](https://user-images.githubusercontent.com/2453371/79673590-02022900-81d3-11ea-9f22-d7eb87153230.png)

Reply:
Thank you for the catch! I had not noticed this. The fix is logical and LGTM.

😸